website/forms.py

```diff
@@ -6,7 +6,7 @@ class IssueEditForm(forms.ModelForm):
 
     class Meta:
         model = Issue
-        fields = ['description', 'screenshot']
+        fields = ['description', 'screenshot', 'label']
```

Comment:
We can set the attributes of the form in the backend. So, if we declare a *required* clause within the form definition in the backend -- It'll make sure that it is required when it is rendered and thus form submission is declined if it is not filled. It also solves the problem that you asked on *Slack*.
https://docs.djangoproject.com/en/1.11/ref/forms/fields/#django.forms.Field.required